[2023-11-23T05:26:50.945+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: rand_user.is_api_available manual__2023-11-23T05:26:47.923275+00:00 [queued]>
[2023-11-23T05:26:50.956+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: rand_user.is_api_available manual__2023-11-23T05:26:47.923275+00:00 [queued]>
[2023-11-23T05:26:50.956+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-11-23T05:26:50.956+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 3
[2023-11-23T05:26:50.957+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-11-23T05:26:50.973+0000] {taskinstance.py:1383} INFO - Executing <Task(HttpSensor): is_api_available> on 2023-11-23 05:26:47.923275+00:00
[2023-11-23T05:26:50.979+0000] {standard_task_runner.py:55} INFO - Started process 3750 to run task
[2023-11-23T05:26:50.984+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'rand_user', 'is_api_available', 'manual__2023-11-23T05:26:47.923275+00:00', '--job-id', '370', '--raw', '--subdir', 'DAGS_FOLDER/predictions.py', '--cfg-path', '/tmp/tmpdnhxnfjn']
[2023-11-23T05:26:50.987+0000] {standard_task_runner.py:83} INFO - Job 370: Subtask is_api_available
[2023-11-23T05:26:51.055+0000] {task_command.py:376} INFO - Running <TaskInstance: rand_user.is_api_available manual__2023-11-23T05:26:47.923275+00:00 [running]> on host f71f95879a41
[2023-11-23T05:26:51.128+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Gaurav
AIRFLOW_CTX_DAG_ID=rand_user
AIRFLOW_CTX_TASK_ID=is_api_available
AIRFLOW_CTX_EXECUTION_DATE=2023-11-23T05:26:47.923275+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-11-23T05:26:47.923275+00:00
[2023-11-23T05:26:51.129+0000] {http.py:120} INFO - Poking: api/
[2023-11-23T05:26:51.139+0000] {base.py:71} INFO - Using connection ID 'rand_user_api' for task execution.
[2023-11-23T05:26:51.141+0000] {http.py:148} INFO - Sending 'GET' to url: https://randomuser.me/api/
[2023-11-23T05:26:52.292+0000] {base.py:213} INFO - Success criteria met. Exiting.
[2023-11-23T05:26:52.329+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=rand_user, task_id=is_api_available, execution_date=20231123T052647, start_date=20231123T052650, end_date=20231123T052652
[2023-11-23T05:26:52.402+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-11-23T05:26:52.457+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
